### ISSUE TITLE
Move AccumuloClient.getInstanceID to InstanceOperations

### ISSUE THREAD

**Issue by mikewalch** [apache/accumulo#907]
AccumuloClient was added to the API for 2.0 so we can make changes (before releasing 2.0.0) without deprecating methods. It has a [getInstanceID](https://github.com/apache/accumulo/blob/d3a5a45ea7530945301884bc7541adb168b05644/core/src/main/java/org/apache/accumulo/core/client/AccumuloClient.java#L270) method that is useful but should be moved to [InstanceOperations](https://github.com/apache/accumulo/blob/b5ea706e8846fdc9374d39be8c95b5a5965b4226/core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java#L25) to minimize the number of methods in AccumuloClient.

Current API:
```java
String instanceID = client.getInstanceID();
```
Proposed API:
```java
String instanceID = client.instanceOperations().getInstanceID();
```
[ClientContext](https://github.com/apache/accumulo/blob/master/core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java) implements AccumuloClient. A lot of internal code calls `ClientContext.getInstanceID`.  It might make sense to add a `getInstanceID` method to ClientContext to limit the amount of internal change:
```
String getInstanceID() {
  return instanceOperations().getInstanceID();
}
```
By adding this method to ClientContext, internal code can still do the following:
```
context.getInstanceID();
```

The current Java doc for the method is:
```
  /**
   * Returns a unique string that identifies this instance of accumulo.
   *
   * @return a UUID
   */
  String getInstanceID();
```

Historically, we have returned a `String` when retrieving the Accumulo instance ID.  However, we say we are returning a UUID.  We should either actually return a Java `UUID` or return a `String` and update the Javadoc to say we are returning a `unique instance ID`.

**Comment by jzgithub1** [comment]
Disregard my last push to my fork.  I will just delete that branch of my fork, redo and resubmit a new pull request.

**Comment by ctubbsii** [comment]
You can do a force push to move your branch (and the pull request) to a commit prior to your last commit. You don't need to delete the branch or resubmit.

**Comment by mikewalch** [comment]
This issues was fixed by PR #926
